refinery/core/models.py

```diff
@@ -2357,15 +2357,21 @@ def render_dataset_visualization_creation(self):
 
     ANALYSIS_CREATION = 'ANALYSIS_CREATION'
 
-    # TODO:
-    # @staticmethod
-    # def record_dataset_analysis_creation():
-    #     event = Event()
-    #     event.save()
-    #     return event
-    #
-    # def render_dataset_analysis_creation(self):
-    #     return '{}'.format(self.user)
+    @staticmethod
+    def record_dataset_analysis_creation(dataset, display_name):
+        user = CuserMiddleware.get_user()
+        blob = json.dumps({
+            'display_name': display_name
+        })
+        event = Event(dataset=dataset, user=user, json=blob,
+                      type=Event.UPDATE, sub_type=Event.ANALYSIS_CREATION)
+        event.save()
+
+    def render_dataset_analysis_creation(self):
+        data = json.loads(self.json)
+        return '{:%x %X}: {} launched analysis {} on data set {}'.format(
```

Comment:
`{:%x %X}` is cool, I'll probably reuse when I trim the timestamp for `Tool's` default `display_names`